deid/dicom/parser.py

```diff
@@ -57,8 +57,17 @@ class DicomParser:
     """
 
     def __init__(
-        self, dicom_file, recipe=None, config=None, force=True, disable_skip=False
-    ):
+        self, dicom_file, recipe=None, config=None, force=True, disable_skip=False):
+        """Create new instance of DicomParser
```

Comment:
```suggestion
        """
        Create new instance of DicomParser
```
Beautiful docstring! My preference is for a newline after the `"""` for the first sentence.